<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostProcessWidget</class>
 <widget class="QWidget" name="PostProcessWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>592</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Post Process</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget_plots">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_angle">
      <property name="font">
       <font>
        <family>Ubuntu</family>
       </font>
      </property>
      <attribute name="title">
       <string>Angle Plots</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="widget_angle_layout"/>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_figures_angles">
         <property name="font">
          <font>
           <family>Ubuntu</family>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_time">
      <property name="font">
       <font>
        <family>Ubuntu</family>
       </font>
      </property>
      <attribute name="title">
       <string>Time Plots</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="widget_time_layout"/>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_figures_time">
         <property name="font">
          <font>
           <family>Ubuntu</family>
          </font>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_rpms">
      <property name="font">
       <font>
        <family>Ubuntu</family>
       </font>
      </property>
      <attribute name="title">
       <string>RPM Plots</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="widget_rpms_layout"/>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_figures_rpms">
         <property name="font">
          <font>
           <family>Ubuntu</family>
          </font>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_cycles">
      <property name="font">
       <font>
        <family>Ubuntu</family>
       </font>
      </property>
      <attribute name="title">
       <string>Cycle Plots</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="widget_cycles_layout"/>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_figures_cycles">
         <property name="font">
          <font>
           <family>Ubuntu</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_space">
      <property name="font">
       <font>
        <family>Ubuntu</family>
       </font>
      </property>
      <attribute name="title">
       <string>Space Plots</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="widget_space_layout"/>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_figures_space">
         <property name="font">
          <font>
           <family>Ubuntu</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_free">
      <property name="font">
       <font>
        <family>Ubuntu</family>
       </font>
      </property>
      <attribute name="title">
       <string>Free Plots</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="widget_free_layout"/>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_figures_free">
         <property name="currentIndex">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>change_plot_widget()</slot>
 </slots>
</ui>
